types/node/url.d.ts

```diff
@@ -46,6 +46,15 @@ declare module "url" {
     interface UrlWithStringQuery extends Url {
         query: string | null;
     }
+    interface FileUrlToPathOptions {
+        /**
+         * `true` if the `path` should be return as a windows filepath, `false` for posix, and `undefined` for the system default.
+         * @default undefined
+         * @since v22.1.0
+         */
+        windows?: boolean | undefined;
+    }
+    interface PathToFileUrlOptions extends FileUrlToPathOptions {}
```

Comment:
```suggestion
    interface PathToFileUrlOptions {
        /**
         * `true` if the path should be treated as a windows filepath, `false` for posix, and `undefined` for the system default.
         * @default undefined
         * @since v22.1.0
         */
        windows?: boolean | undefined;
    }
```
I know that the interface was defined this way pre-reversion, but there's no reason why the two functions should always keep the same options going forward, and the docblock semantics are different. Defining each interface individually seems like a better approach.